src/mocks/handlers.ts

```diff
@@ -65,7 +66,12 @@ export const handlers = [
         parent_id: null,
       };
     }
-
+    body = {
+      ...body,
+      catalogue_item_properties: body.catalogue_item_properties?.map(
+        (property) => ({ ...property, id: generateUniqueId('test_id') })
```

Comment:
```suggestion
        (property) => ({ ...property, id: generateUniqueId('test_id_') })
```
For consistency with the other uses of `generateUniqueId`.